packages/types-kit/src/types.ts

```diff
@@ -7,6 +7,15 @@ export enum OperationType {
   DelegateCall // 1
 }
 
+export const SignatureTypes = {
```

Comment:
regarding `enum` vs `const enum` vs `const`: 
https://www.perplexity.ai/search/enum-vs-const-enum-in-typescri-bBwicw7YQDGRvdxEiM56zA#0

here's why I went with const:

> Type safety
>     Runtime object availability
>     No extra generated code
>     Better compatibility with modern JavaScript practices[6](https://sergiocarracedo.es/typescript-enums/)